core/src/test/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceServiceSpec.scala

```diff
@@ -1388,12 +1395,14 @@ class WorkspaceServiceSpec
       Map.empty
     )
     when(services.workspaceManagerDAO.getWorkspace(any[UUID], any[RawlsRequestContext])).thenReturn(
-      new WorkspaceDescription().azureContext(
-        new AzureContext()
-          .tenantId("fake_tenant_id")
-          .subscriptionId("fake_sub_id")
-          .resourceGroupId("fake_mrg_id")
-      )
+      new WorkspaceDescription()
+        .stage(WorkspaceStageModel.MC_WORKSPACE)
```

Comment:
@aherbst-broad I had to introduce `stage` in a bunch of different spots throughout this file (and various mocks), which is making me a little nervous about the potential scope of impact of this change.  Looking for validation that this is _ok_ and makes things reflect the expected reality, rather than me shoving an assumption where it doesn't belong.